### ISSUE TITLE
How to validate only dropdown search field?

### ISSUE THREAD

**Issue by ffd114** [salim-lachdhaf/dropdown_search#732]
I run into issue where I need to validate only the dropdown search field, usually when using `TextFormField` I can just set the `key` with `GlobalKey<FormFieldState>()`, but I cannot find the way to do this on dropdown search, is there a way to to validate only the drop down search field?

**Comment by salim-lachdhaf** [comment]
yes, in the same way as the `TextFormField` .
just add a key to your DropdownSearch  like :

`final myKey  = GlobalKey<DropdownSearchState<T>>`();

then call 
`myKey.currentState?.validate()`

**Comment by ffd114** [comment]
But `DropdownSearchState` does not have `validate()` function, so it throws error on me. Anyway for now I just forked this package and add some access to inner `FormField` key, even though it is not very clean but for now it's sufficient.

Thanks for this awesome package!

**Comment by salim-lachdhaf** [comment]
Yes, you are right. you can only change `validationMode`.
But why did not put your `DropdownSearch` inside a `Form`  and call validate on form that contains your `DropdownSearch` ?

**Comment by ffd114** [comment]
In my use case, I use `DropdownSearch` to list foreign currencies, below the `DropdownSearch` there are text field to fill the exchange rate and a button to fetch the exchange rate from my server. When I click this button, I need to validate only the selected currency from `DropdownSearch`, if it is valid, it will call the API and the text field will be filled with the data from my server. If I use form valid, it will also check another field unrelated for my API calls

**Comment by salim-lachdhaf** [comment]
no, i meant you create a `Form` only for `DropdownSearch` Which will give you a structure like this

```
Form
--TextFormField1
--TextFormField2
--TextFormField..
--Form 
----DropdownSearch
--TextFormFieldx
...
```

**Comment by ffd114** [comment]
Ahh sorry I misunderstood your question, I haven't tried it that way and did not know if that will works. Thanks for the insight!

But for now even if it works, I think I will stick with the current implementation, since I still need to access the `DropdownSearchState` key and by using Form key I need to declare another key
